fastlane/Fastfile

```diff
@@ -628,6 +628,24 @@ REPOSITORY_NAME="WordPress-Android"
     "#{build_dir}#{name}"
   end
 
+  #####################################################################################
+    # download_translations
+    # -----------------------------------------------------------------------------------
+    # This lane download the string translations from GlotPress
+    # -----------------------------------------------------------------------------------
+    # Usage:
+    # bundle exec fastlane download_translations
+    #####################################################################################
+  lane :download_translations do |options|
+    # Adapt the GP project URL and lint_task as needed if testing a project other than WordPress
+    android_download_translations(
+      res_dir: File.join(ENV['PROJECT_NAME'], 'src', 'main', 'res'),
+      glotpress_url: 'https://translate.wordpress.org/projects/apps/android/dev/',
+      locales: SUPPORTED_LOCALES,
+      lint_task: 'lintWordpressVanillaRelease'
```

Comment:
🤔  Before that change, the legacy action was using `lintVanillaRelease` (it read that lint task name from the `ENV["validate_translations"]="lintVanillaRelease"` env var defined on line 58… and which can actually get removed while we're at it, because it wasn't used by anything other than the legacy action that you just replaced 😉

Reply:
Removed the env var 👍